pkg/ngrokgodriver/driver.go

```diff
@@ -0,0 +1,121 @@
+package ngrokgodriver
+
+import (
+	"context"
+	"io"
+	"log"
+	"net"
+	"reflect"
+
+	"golang.ngrok.com/ngrok"
+	"golang.ngrok.com/ngrok/config"
+	"golang.org/x/sync/errgroup"
+)
+
+type TunnelManager struct {
+	session ngrok.Session
+	tunnels map[string]ngrok.Tunnel
+}
+
+func NewTunnelManager() (*TunnelManager, error) {
+	opts := []ngrok.ConnectOption{
+		ngrok.WithAuthtokenFromEnv(),
+	}
```

Comment:
What do the logs look like with how you have it? I'm wondering if we need to set it up to use our logger https://github.com/ngrok/ngrok-go/blob/45a13036a1ff/session.go#L222

Reply:
```
1.6716860153547833e+09	DEBUG	events	Normal	{"object": {"kind":"Ingress","namespace":"default","name":"test-app-1","uid":"ef836330-9c76-4739-bd13-5660ef8e65a9","apiVersion":"networking.k8s.io/v1","resourceVersion":"11469545"}, "reason": "CreatedEdge", "message": "Created edge edghts_<redacted>"}
1.6716860154495308e+09	DEBUG	events	Warning	{"object": {"kind":"Ingress","namespace":"default","name":"test-app-2","uid":"0123939d-f9cc-411a-8070-57b6fd83867e","apiVersion":"networking.k8s.io/v1","resourceVersion":"11469294"}, "reason": "EdgeExists", "message": "Edge already exists"}
1.6716860155462549e+09	DEBUG	events	Normal	{"object": {"kind":"Ingress","namespace":"default","name":"test-app-2","uid":"0123939d-f9cc-411a-8070-57b6fd83867e","apiVersion":"networking.k8s.io/v1","resourceVersion":"11469294"}, "reason": "CreatedEdge", "message": "Created edge edghts_<redacted>"}
2022/12/22 22:44:19 Accepted connection from  <redacted>:59426
2022/12/22 22:49:22 Connection closed:  <nil>
2022/12/23 03:52:49 Accepted connection from <redacted>:46789
2022/12/23 03:57:49 Connection closed:  <nil>
```

Yeah, I'll update it to pass the logger in since having 2 different formats in there isn't great

Reply:
Fixed in https://github.com/ngrok/ngrok-ingress-controller/pull/85/commits/d320396e9a92e568b567762b17ea16021a1927b2